stable/redis/templates/redis-master-statefulset.yaml

```diff
@@ -135,6 +135,16 @@ spec:
           mountPath: /opt/bitnami/redis/etc/redis.conf
           subPath: redis.conf
         {{- end }}
+      {{- if and .Values.master.persistence.enabled (not .Values.persistence.existingClaim) | and .Values.master.securityContext.enabled }}
+      initContainers:
+      - name: redis-init
+        image: alpine:3.8
```

Comment:
This should be configurable depending on the `values.yaml`:
```
        image: "{{ .Values.volumePermissions.image.name }}:{{ .Values.volumePermissions.image.tag }}"
        imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
```

Reply:
other image structure do not have filed "name", but have "repository"
and use template helpers